src/Mod/Path/PathScripts/PathProfile.py

```diff
@@ -482,15 +482,22 @@ def areaOpShapes(self, obj):
                             custDepthparams = self.depthparams
                             self._addDebugObject('Rotation_Indiv_Shp', shape)
 
-                            if self.expandProfile:
-                                shapeEnv = self._getExpandedProfileEnvelope(obj, shape, False, obj.StartDepth.Value, finalDep)
+                            try:
+                                if self.expandProfile:
+                                    shapeEnv = self._getExpandedProfileEnvelope(obj, shape, False, obj.StartDepth.Value, finalDep)
+                                else:
+                                    shapeEnv = PathUtils.getEnvelope(shape, depthparams=custDepthparams)
+                            except Exception as ee:
+                                # check if multiple features selected
+                                if obj.Base and len(obj.Base[0][1]) > 1:
+                                    PathLog.error(ee)
+                                    msg = translate('PathProfile', 'Please consider setting "Handle Multiple Features" to "Collectively" and recomputing.')
+                                    PathLog.error('\n' + msg)
```

Comment:
isn't it customary to have the `\n` as the last character of a message?

Reply:
> isn't it customary to have the `\n` as the last character of a message?

As some customs are.  This is a custom, purposeful usage.  I will change it to accommodate merge requirements.

Reply:
Changed.